.github/workflows/ci.yml

```diff
@@ -31,3 +31,8 @@ jobs:
         run: |
           sudo apt-get install sqlite3
           touch ./dev.sqlite3
+
+      - name: Run migrations with SQLite
```

Comment:
You should create another job in this file. Because the current job title is `lint-and-format`.